src/compilerWrapper.ts

```diff
@@ -106,9 +107,10 @@ export function compile(
 	const outputFiles = {};
 	try {
 		const sourceContent = source.content !== undefined ? source.content : readFileSync(sourcePath, 'utf8');
-		const cmdPrefix = settings.cmdPrefix || `npx ${npxArg} scryptc${settings.scVersion ? '@' + settings.scVersion : ''}`;
+		const cmdPrefix = settings.cmdPrefix || getDefaultsCryptc();
 		const cmd = `${cmdPrefix} compile ${settings.asm || settings.desc ? '--asm' : ''} ${settings.ast || settings.desc ? '--ast' : ''} ${settings.debug == false ? '' : '--debug'} -r -o "${outputDir}" ${settings.cmdArgs ? settings.cmdArgs : ''}`;
-		const output = execSync(cmd, { input: sourceContent, cwd: curWorkingDir }).toString();
+		let output = execSync(cmd, { input: sourceContent, cwd: curWorkingDir }).toString();
+		output = output.split(/\r?\n/g).join('\n');
```

Comment:
add a comment explaining why this is needed

Reply:
added

Reply:
see d4172b71e62e4f3b68a150aea78babf5e54f3d39